src/json.hpp

```diff
@@ -1241,15 +1336,14 @@ class basic_json
 
     @sa @ref basic_json(const typename string_t::value_type*) -- create a
     string value from a character pointer
-    @sa @ref basic_json(const CompatibleStringType&) -- create a string value
+    @sa @ref basic_json(const CompatibleStringType&) -- create a string
+    value
```

Comment:
nitpick: you should probably revert all unnecessary formatting changes, since they make going through the relevant parts of the diffs harder